lib/Sema/TypeCheckAttr.cpp

```diff
@@ -2464,6 +2464,10 @@ void TypeChecker::addImplicitDynamicAttribute(Decl *D) {
     return;
 
   if (auto *VD = dyn_cast<VarDecl>(D)) {
+    // Don't turn stored into computed properties. This could conflict with
+    // exclusivity checking.
+    if (VD->hasStorage())
+      return;
```

Comment:
This seems wrong to me. Why are we calling `addImplicitDynamicAttribute` on a stored property in the first place?

Reply:
I think `addImplicitDynamicAttribute` should be the one to include the check for `decl->getModuleContext()->isImplicitDynamicEnabled()`, then. But even without that, I don't understand why it's okay to make the whole module implicitly dynamic "except for stored properties". Yes, it would violate exclusivity, but that just means "make the whole module implicitly dynamic" isn't going to work.